website/docs/terms/monotonically-increasing.md

```diff
@@ -0,0 +1,11 @@
+---
+id: monotonically-increasing
+title: Monotonically Increasing
+description: Monotonicity means unchanging (think monotone). A monotonically-increasing value is a value which increases at a constant rate, for example the values 1, 2, 3, 4.
```

Comment:
@KiraFuruichi what is the difference between a description and a hoverSnippet? I had a glance at a few terms' files and they seemed to be basically the same

Reply:
For the most part, the will be identical. The description is the page description which is important for SEO/ranking purposes. The hoverSnippet is what displays when a user hover overs the linked term on another page (ie a basic definition of the term).